kafka-ui-react-app/src/components/Topics/List/ListItem.tsx

```diff
@@ -109,25 +109,35 @@ const ListItem: React.FC<ListItemProps> = ({
         <BytesFormatted value={segmentSize} />
       </td>
       <td className="topic-action-block" style={{ width: '4%' }}>
-        {!internal && !isReadOnly && vElipsisVisble ? (
-          <div className="has-text-right">
-            <Dropdown label={<VerticalElipsisIcon />} right>
-              {cleanUpPolicy === 'DELETE' && (
-                <DropdownItem onClick={clearTopicMessagesHandler} danger>
-                  Clear Messages
-                </DropdownItem>
-              )}
-              {isTopicDeletionAllowed && (
-                <DropdownItem
-                  onClick={() => setDeleteTopicConfirmationVisible(true)}
-                  danger
-                >
-                  Remove Topic
-                </DropdownItem>
-              )}
-            </Dropdown>
-          </div>
-        ) : null}
+        <div
+          className="has-text-right"
```

Comment:
I think it will be better to use styledComponents and control styles by passing props instead of classNames and inline CSS